tests/hardhat/Governance/GovernanceBravo/queueTest.ts

```diff
@@ -0,0 +1,179 @@
+import { BigNumber, ContractTransaction, Signer } from "ethers";
+import { ethers, network } from "hardhat";
+import { loadFixture, mine } from "@nomicfoundation/hardhat-network-helpers";
+import { smock, MockContract, FakeContract } from "@defi-wonderland/smock";
+import chai from "chai";
+const { expect } = chai;
+chai.use(smock.matchers);
+
+const { encodeParameters } = require("../../../Utils/BSC");
+
+import {
+  GovernorBravoDelegate,
+  GovernorBravoDelegate__factory,
+  XVSVault,
+  XVSStore,
+  XVS,
+  Timelock,
+} from "../../../../typechain";
+import { ProposalState, ProposalType } from "../../util/Proposals";
+import { convertToUnit } from "../../../../helpers/utils";
+
+let root: Signer;
+let customer: Signer;
+let accounts: Signer[];
+let governorBravoDelegate: MockContract<GovernorBravoDelegate>;
+let xvsVault: FakeContract<XVSVault>;
+let xvsStore: FakeContract<XVSStore>;
+let xvsToken: FakeContract<XVS>;
+let timelock: FakeContract<Timelock>;
+
+type GovernorBravoDelegateFixture = {
+  governorBravoDelegate: MockContract<GovernorBravoDelegate>;
+  xvsVault: FakeContract<XVSVault>;
+  xvsStore: FakeContract<XVSStore>;
+  xvsToken: FakeContract<XVS>;
+  timelock: FakeContract<Timelock>;
+};
+
+async function governorBravoFixture(): Promise<GovernorBravoDelegateFixture> {
+  const GovernorBravoDelegateFactory =
+    await smock.mock<GovernorBravoDelegate__factory>("GovernorBravoDelegate");
+  const governorBravoDelegate = await GovernorBravoDelegateFactory.deploy();
+  const xvsVault = await smock.fake<XVSVault>("XVSVault");
+  const xvsStore = await smock.fake<XVSStore>("XVSStore");
+  const xvsToken = await smock.fake<XVS>("XVS");
+  const timelock = await smock.fake<Timelock>("Timelock");
+  return { governorBravoDelegate, xvsVault, xvsStore, xvsToken, timelock };
+}
+
+const proposalConfigs = {
```

Comment:
Since all tests share this does it make sense to put this in its own file `tests/hardhat/Governance/GovernanceBravo` so all tests share the same config and get updated together?

Reply:
yeah this could be one optimisation... We agreed that the initial migration will consist only of "translating" the tests from saddle to hardhat, since they are a lot and we do not have the time to migrate + optimise. But will note this and when it comes time for optimising the tests we will involve you to give suggestions as well :)